modules/gltf/doc_classes/GLTFObjectModelProperty.xml

```diff
@@ -81,7 +81,7 @@
 			Unknown or not set object model type. If the object model type is set to this value, the real type still needs to be determined.
 		</constant>
 		<constant name="GLTF_OBJECT_MODEL_TYPE_BOOL" value="1" enum="GLTFObjectModelType">
-			Object model type "bool". Represented in the glTF JSON as a boolean, and encoded in a [GLTFAccessor] as "SCALAR". When encoded in an accessor, a value of 0 is false, and any other value is true.
+			Object model type "bool". Represented in the glTF JSON as a boolean, and encoded in a [GLTFAccessor] as "SCALAR". When encoded in an accessor, a value of [code]0[/code] is [code]false[/code], and any other value is [code]true[/code].
```

Comment:
Are either "bool" or "SCALAR" literals that should also be wrapped in [code]? Not familiar with the GLTF classes. Usually quotes is a sign that the value might be a literal and should maybe be wrapped in [code]

"bool" probably shouldn't be, because all the other constants also use quotes

Reply:
There's more constants around this, hence I decided to not change the quotes for, at least for now, consistency with the rest of the enum.